model/host/db.go

```diff
@@ -235,12 +235,23 @@ func runningHostsQuery(distroID string) bson.M {
 }
 
 // byRunningStatusQuery produces a query that returns all hosts
-// with the running status that belong to the given distro.
+// with the running status that belong to the given distro and are
+// started by Evergreen.
 func byRunningStatusQuery(distroID string) bson.M {
```

Comment:
Might make sense to rename this since it's not really about hosts literally in the "running" state, but rather hosts that are capable of accepting new tasks to run.